README.md

```diff
@@ -228,13 +228,13 @@ A common scenario of processing actions is:
 Using sendToWatson to update context simplifies the bot code compared to solution using updateContext below.
 
 ```js
-const checkBalance =  (context) => new Promise((resolve, reject) => {
+const checkBalance =  async (context) => {
   //do something real here
   const contextDelta = {
     validAccount: true,
     accountBalance: 95.33
   };
-  resolve(context);
+  await Promise.resolve(context);
```

Comment:
This code can be further simplified to `return context` - async turns all returns to promises.